src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public Integer getQuantity(String fruit) {
+        return fruits.get(fruit);
+    }
+
+    public void put(String fruit, Integer quantity) {

```

Comment:
The method 'put' provides direct access to the data. It's advisable to implement validation checks before adding or updating data in the map.

Reply:
ok, done